shared/chat/conversation-list/conversation-list.js

```diff
@@ -81,8 +87,23 @@ const ConversationList = (props: Props) => {
   }
 
   return (
-    <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={styles.container}>
-      {!!props.filter && <ConversationFilterInput style={styles.filter} {...props.filter} />}
+    <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true}>
+      {!!props.filter && (
+        <ConversationFilterInput
+          onBack={props.onBack}
+          style={styles.filter}
+          isLoading={props.filter.isLoading}
+          filter={props.filter.filter}
+          onSetFilter={props.filter.onSetFilter}
+          filterFocusCount={0}
+          noShortcut={true}
+          onBlur={() => {}}
```

Comment:
is it safe to just have these do nothing now? why?

Reply:
I actually just didn't know what they were for, and this isn't used in chat at all, just in the attachment popup I'm building now, so I left them like that. But I guess I'll just figure out and do them right here.

Reply:
OK I don't think we need these for choose-conversation, but I added focusOnMount for the popup.